js/popup.js

```diff
@@ -0,0 +1,46 @@
+import { createProposals, Types } from './data.js';
+
+const mapCanvas = document.querySelector('#map-canvas');
+
+const cardTemaplate = document.querySelector('#card').content.querySelector('.popup');
```

Comment:
```suggestion
const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
```

поставь проверку орфографии

Reply:
done